app/views/users/index.html.erb

```diff
@@ -1,9 +1,5 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Users List</title>
-  </head>
-  <body>
-    <h1>A list of all users</h1>
-  </body>
-</html>
\ No newline at end of file
+<div class="flex flex-col w-full gap-y-5">
+  <% @users.each do |user| %>
+    <%= render partial: 'shared/user', locals: {user: user} %>
+  <% end %>
+</div>
```

Comment:
- [ ] Please fix the layout for the user to match the given wireframe. Which you can arrange with flexbox. User photo, username should be on the same line watching the space between them. The post counter should be in the right bottom.
![image](https://github.com/rmiyoyo/blog-app/assets/91486791/38996fd4-8e0c-45d7-84a8-b8a986a9de0a)
![image](https://github.com/rmiyoyo/blog-app/assets/91486791/d0cf7155-ed32-4e3b-a230-9b4785ef1312)